core/arch/arm/kernel/secure_partition.c

```diff
@@ -422,6 +379,196 @@ static TEE_Result sp_dt_get_uuid(const void *fdt, int node,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result sp_is_elf_format(const void *fdt, int sp_node,
+				   bool *is_elf_format)
+{
+	TEE_Result res = TEE_SUCCESS;
+	uint32_t elf_format = 0;
+
+	res = sp_dt_get_u32(fdt, sp_node, "legacy-elf-format", &elf_format);
```

Comment:
Why not just "elf-format"?

Reply:
The main reason was to emphasize that the ELF format is not the preferable one anymore. I you would prefer "elf-format" I can change it.

Reply:
I would prefer "elf-format" as that's more neutral. But that's just my opinion if it's important to emphasize that this is less desired even in the name of the variable keep it as it is.

Reply:
Changed to "elf-format".